src/com/customify/cli/components/Button.java

```diff
@@ -0,0 +1,4 @@
+package com.customify.client.components;
+
+public class Button {
```

Comment:
I'm not sure why to added an empty class

Reply:
### Here are some screenshots
1. In the Code editor
![image](https://user-images.githubusercontent.com/61510549/120077086-e7be9e00-c0a8-11eb-9ecb-ca88e0a138d9.png)
2. Here is the response in the database
![image](https://user-images.githubusercontent.com/61510549/120077109-06249980-c0a9-11eb-8c55-002fd4e01934.png)